diff --git a/node_modules/decode-uri-component/index.js b/node_modules/decode-uri-component/index.js
index 501f80d..3351a5f 100644
--- a/node_modules/decode-uri-component/index.js
+++ b/node_modules/decode-uri-component/index.js
@@ -157,7 +157,7 @@ function customDecodeURIComponent(input) {
 	return input;
 }
 
-export default function decodeUriComponent(encodedURI) {
+module.exports = function decodeUriComponent(encodedURI) {
 	if (typeof encodedURI !== 'string') {
 		throw new TypeError(`Expected \`encodedURI\` to be of type \`string\`, got \`${typeof encodedURI}\``);
 	}
diff --git a/node_modules/decode-uri-component/package.json b/node_modules/decode-uri-component/package.json
index 8d01f06..9ffe115 100644
--- a/node_modules/decode-uri-component/package.json
+++ b/node_modules/decode-uri-component/package.json
@@ -10,7 +10,7 @@
 		"email": "sam.verschueren@gmail.com",
 		"url": "github.com/SamVerschueren"
 	},
-	"type": "module",
+	"type": "commonjs",
 	"exports": {
 		"types": "./index.d.ts",
 		"default": "./index.js"
